plugins/event_notification/providers/email/lib/batch/recipients/KEmailNotificationCategoryRecipientEngine.php

```diff
@@ -7,43 +7,44 @@
  */
 class KEmailNotificationCategoryRecipientEngine extends KEmailNotificationRecipientEngine
 {
-	/* (non-PHPdoc)
+	/** (non-PHPdoc)
 	 * @see KEmailNotificationRecipientEngine::getRecipients()
-	 */ 
+	 **/
 	function getRecipients(array $contentParameters)
 	{
 		$recipients = array();
 		$pager = new KalturaFilterPager();
-		$pager->pageSize = 500;
+		$pager->pageSize = 100;
 		$pager->pageIndex = 1;
-		$categoryUserIds = array();
-		$maxPagesToScan = 2;
+		$userPager = new KalturaFilterPager();
+		$userPager->pageSize = $pager->pageSize;
+		$maxPagesToScan = 7;
```

Comment:
The former procedure tried to handle 1000 users, why block at 700

Reply:
fixed to 1050